Sakura/AI/response.py

```
@@ -1,40 +1,32 @@
-    if state.openrouter_client:
-        log_action("INFO", "🤖 Trying OpenRouter API...", user_info)
-        try:
-            response = await openrouter_response(user_message, user_name, user_info, user_id, image_bytes)
-            if response:
-                source_api = "OpenRouter"
-                log_action("INFO", f"✅ {source_api} response generated: '{response[:50]}...'", user_info)
-        except Exception as e:
-            log_action("ERROR", f"❌ OpenRouter API error: {e}. Falling back to Gemini.", user_info)
+async def get_response(
+    user_message: str,
+    user_name: str,
```

Ruff (0.13.2)

Unused function argument: `user_name`

(ARG001)